### PR TITLE
Highlight homepage title, add authors to page header #25

### PR DESCRIPTION
**Pull request by DrDrij**
This isn't ready to merge. Needs a test build. I'm trying to get my local environment up.

But I've added styles and a template for the header and authors.

Proposed layout:

image

image

### CODE REVIEW

**Comment by mmcky** [comment]
thanks @DrDrij I will build this in the morning

**Comment by mmcky** [comment]
@DrDrij I have setup a PR on `lecture-python-programming` to test this branch as builds are done on `master`.

We should setup a sample project build for this REPO as a `CI` test. I'll try and do that soon.

**Comment by mmcky** [comment]
https://github.com/QuantEcon/lecture-python-programming.myst/pull/68

**Comment by mmcky** [comment]
@DrDrij seems to be an issue with `scss`

```
Extension error:
Handler <function add_static_path at 0x7f4d00800820> for event 'builder-inited' threw an exception
Traceback (most recent call last):
  File "/usr/share/miniconda3/envs/qe-lectures/lib/python3.8/site-packages/sphinx/events.py", line 110, in emit
    results.append(listener.handler(self.app, *args))
  File "/usr/share/miniconda3/envs/qe-lectures/lib/python3.8/site-packages/quantecon_book_theme-0.0.1-py3.8.egg/quantecon_book_theme/__init__.py", line 34, in add_static_path
    sass_compile(dirname=(source_dir, output_dir), output_style="compressed")
  File "/usr/share/miniconda3/envs/qe-lectures/lib/python3.8/site-packages/libsass-0.20.1-py3.8-linux-x86_64.egg/sass.py", line 738, in compile
    raise CompileError(v)
sass.CompileError: Error: Top-level selectors may not contain the parent selector "&".
        on line 429 of ../../../../../usr/share/miniconda3/envs/qe-lectures/lib/python3.8/site-packages/quantecon_book_theme-0.0.1-py3.8.egg/quantecon_book_theme/scss/quantecon-book-theme.scss
>> .main-index & {

   ^


The above exception was the direct cause of the following exception:
```

**Comment by mmcky** [comment]
- [ ] add CI to test project builds for `theme` PR's.

**Comment by mmcky** [comment]
This project is building on the test PR for lecture-python-programming.myst now

https://5f7fac05c8d142177e3854fa--epic-agnesi-957267.netlify.app/intro.html

**Comment by mmcky** [comment]
@DrDrij builds are now working on this branch for you so you can see results of any changes.

File: quantecon_book_theme/layout.html [hunk 1]
```diff
@@ -22,7 +22,7 @@
 {% block sidebarsourcelink %}{% endblock %}
 
 {% block body_tag %}
-<body>
+<body{% if master_doc == 'index' %} class="main-index"{% endif %}>
```
Comment (mmcky) [inline quantecon_book_theme/layout.html:25]: @AakashGfude in the same build `master_doc` is called `intro`. Is there a way to check if a current page is the `master_doc` (as a boolean condition?)

Comment (mmcky) [reply quantecon_book_theme/layout.html:25]:
```
if <filename> == master_doc
```

**Comment by mmcky** [comment]
Looking good @DrDrij and @AakashGfude 🥳

**Comment by mmcky** [comment]
@AakashGfude @DrDrij can this be merged?

**Comment by AakashGfude** [comment]
@mmcky yes I think so. But will wait for @DrDrij 's cue.

**Comment by mmcky** [comment]
- [ ] once this is merged @mmcky to update all the `CI` for this and `lecture-python-programming.myst`

**Comment by AakashGfude** [comment]
wait @mmcky this needs one more small fix. I should check for the exact string of master_doc instead of just checking if master_doc name is a part of the pagename

**Comment by AakashGfude** [comment]
all good now 😊

**Comment by mmcky** [comment]
thanks @AakashGfude and @DrDrij -- love your work. I will merge this and then update `CI`.

**Pull request merged**